plugins/org.locationtech.udig.catalog.jgrass/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,8 @@ Require-Bundle: org.eclipse.ui,
  org.locationtech.udig.catalog.ui,
  org.locationtech.udig.project.ui,
  org.locationtech.udig.mapgraphic,
- org.locationtech.udig.style
+ org.locationtech.udig.style,
+ org.junit
```

Comment:
Not a good idea to depent on a test-library at runtime ;)

What is the reason to add it?

Reply:
Was required to have JUnit available for the test case, but I moved the test case to the test bundle - therefore not required anymore :)